testdata/COUNTERS:Ethernet68:Queues_alias.txt

```diff
@@ -0,0 +1 @@
+{"Ethernet68/1:0":{},"Ethernet68/1:1":{"SAI_QUEUE_STAT_BYTES":"0","SAI_QUEUE_STAT_DROPPED_BYTES":"0","SAI_QUEUE_STAT_DROPPED_PACKETS":"0","SAI_QUEUE_STAT_PACKETS":"0"},"Ethernet68/1:10":{},"Ethernet68/1:11":{},"Ethernet68/1:12":{},"Ethernet68/1:13":{},"Ethernet68/1:14":{},"Ethernet68/1:15":{},"Ethernet68/1:16":{},"Ethernet68/1:17":{},"Ethernet68/1:18":{},"Ethernet68/1:19":{},"Ethernet68/1:2":{},"Ethernet68/1:3":{},"Ethernet68/1:4":{},"Ethernet68/1:5":{},"Ethernet68/1:6":{},"Ethernet68/1:7":{},"Ethernet68/1:8":{},"Ethernet68/1:9":{}}
```

Comment:
>{"Ethernet68/1:0":{},"Ethernet68/ [](start = 0, length = 33)

can you beautify them? hard to read in oneline.

Reply:
JSON files are pretty-printed.